Update rich-presence.md

pandakar-gaming opened the pull request:
Put Condition Syntax into its own little subsection with a proper header and a bit more description as to what the links in that section refer to.

I don't know about other developers, but I've been finding that it's not immediately clear when reviewing this page as to where the syntax definitions are. This modification is made in the hope that it can at least make the Condition Syntax jump out a bit more on the page.

wescopeland commented:
Correct me if I'm wrong - aren't the docs themselves a compile target of this repo's wiki?

christianlegge commented:
That explains a lot! Sorry, will delete and edit there. I should have figured there was a reason there were no other forks..

wescopeland commented:
No worries - forking and opening a PR is a perfectly reasonable assumption and what we ask for our other docs projects such as [api-docs](https://github.com/RetroAchievements/api-docs).

pandakar-gaming committed:
Update rich-presence.md

Put Condition Syntax into its own little subsection with a proper header and a bit more description as to what the links in that section refer to.

pandakar-gaming committed:
Update rich-presence.md

Add missing '/' in newly added links
